cmd/snap-confine/snap-confine.c

```diff
@@ -96,6 +96,42 @@ static void sc_maybe_fixup_udev(void)
 	}
 }
 
+/**
+ * sc_preservation_context remembers clobbered state to restore.
```

Comment:
that's a very abstract name.

sc_preserved_process_state

Reply:
I will rename this as follows:

- `sc_preserved_process_state`
- `sc_preserve_and_mangle_process_state`
- `sc_restore_process_state`

How does that sound?

Reply:
looks good, except I would go for:

sc_preserve_and_mangle_process_state =>  sc_preserve_and_sanitize_process_state

Reply:
Thanks